src/components/buildArguments.ts

```diff
@@ -12,31 +12,27 @@ export const buildArguments = (
 	storageUri: Uri,
 ) => {
 	const { command } = message;
+	const args: string[] = [];
+
+	const codemodArguments =
+		command.kind !== 'executeLocalCodemod'
+			? (command.arguments ?? []).flatMap(({ name, value }) => [
+					`--arg:${name}`,
+					String(value),
+			  ])
+			: [];
+
+	args.push(...codemodArguments);
```

Comment:
I think that won't work, these arguments must be passed as the last ones because the first argument must be the codemod name (the positional argument)